api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -88,6 +97,14 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
           if (workspace.getCdrVersion() != null) {
             result.setCdrVersionId(String.valueOf(workspace.getCdrVersion().getCdrVersionId()));
           }
+
+          ArrayList<org.pmiops.workbench.db.model.WorkspaceUserRole> usersOnWorkspace = new ArrayList<org.pmiops.workbench.db.model.WorkspaceUserRole>();
+          Iterator<org.pmiops.workbench.db.model.WorkspaceUserRole> iter = workspace.getWorkspaceUserRoles().iterator();
```

Comment:
I don't think you need to construct usersOnWorkspace here... just use workspace.getWorkspaceUserRoles().stream() below.